src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Utils/SetupUtil.cs

```diff
@@ -49,7 +53,16 @@ public static HttpClient GetTestClient(CustomWebApplicationFactory<Altinn.App.St
                     services.AddSingleton<IRegister, RegisterMockSI>();
                     services.AddSingleton<Altinn.Common.PEP.Interfaces.IPDP, PepWithPDPAuthorizationMockSI>();
                     services.AddSingleton<IApplication, ApplicationMockSI>();
-                    services.AddSingleton<IAltinnApp, AltinnApp>();
+                    if (customValidation)
+                    {
+                        services.AddSingleton<IAltinnApp, IntegrationTests.Mocks.Apps.tdd.custom_validation.AltinnApp>();
+                    }
+                    else
+                    {
+                        services.AddSingleton<IAltinnApp, IntegrationTests.Mocks.Apps.tdd.endring_av_navn.AltinnApp>();
```

Comment:
Vi bør kanskje endre interfacet til å ta en app type fra testen. Men det kan vi ta i en annen endring. Ser for meg at vi få rmange apps etterhvert.

Reply:
Ja, lurte litt på hvordan jeg skulle løse det, så det er sikkert noe vi bør se på.